src/development/ui/navigation/deep-linking.md

```diff
@@ -55,7 +55,18 @@ host name matches the name defined in `AndroidManifest.xml`:
 ```
 adb shell am start -a android.intent.action.VIEW \
     -c android.intent.category.BROWSABLE \
-    -d "http://flutterbooksample.com/book/1"
+    -d "http://flutterbooksample.com/book/1" \
+    <package name>
+```
+
+Replace the `<package name>` with the package name of your android app. If the package
+name is `com.example.myflutterapp`, the command will then be:
```

Comment:
**issue:** Need to capitalize Android and use active voice in the second sentence.
```suggestion
Replace the `<package name>` with the package name of your Android app.
If you named the package `com.example.myflutterapp`, run the following command:
```